src/fastcs/attributes.py

```diff
@@ -49,12 +49,10 @@ class AttrHandlerRW(AttrHandlerR, AttrHandlerW):
 class SimpleAttrHandler(AttrHandlerRW):
     """Handler for internal parameters"""
 
-    async def put(self, attr: AttrW[T], value: T) -> None:
+    async def put(self, attr: AttrRW[T], value: T) -> None:  # type: ignore
```

Comment:
Ah, OK. We can't remove the `isinstance` because `put` needs to take an `AttrW`. Let's keep it
as it was.